Sources/SwiftBasicFormat/BasicFormat.swift

```diff
@@ -237,6 +236,13 @@ open class BasicFormat: SyntaxRewriter {
       (_, nil),
       (nil, _):
       return false
+    case (_, .colon):
+      let isTernaryExpr = second?.parent?.is(UnresolvedTernaryExprSyntax.self) == true
+      || second?.parent?.is(TernaryExprSyntax.self) == true
+
+      if !isTernaryExpr {
+        return false
+      }
```

Comment:
You can write this slightly  shorter by using `keyPathInParent`, just like we do in the second `switch` statement below.

```suggestion
    case (_, .colon):
      if second?.keyPathInParent != \TernaryExprSyntax.colonMark
        && second?.keyPathInParent != \UnresolvedTernaryExprSyntax.colonMark
      {
        return false
      }
```

Reply:
Great! Thank you 👍